src/app/workspace/resource/permission-info/permission-info.component.ts

```diff
@@ -0,0 +1,184 @@
+import { ConnectionPositionPair, OriginConnectionPosition, Overlay, OverlayConnectionPosition, OverlayPositionBuilder, ScrollStrategy, ScrollStrategyOptions } from '@angular/cdk/overlay';
+import { Component, ElementRef, Input, OnInit, ViewChild } from '@angular/core';
+import { PermissionUtil } from '@dasch-swiss/dsp-js';
+
+export interface PermissionObj {
+    name: string;
+    label: string;
+    icon: string;
+}
+
+export interface PermissionGroup {
+    group: string;
+    restriction: PermissionUtil.Permissions[];
+}
+
+@Component({
+    selector: 'app-permission-info',
+    templateUrl: './permission-info.component.html',
+    styleUrls: ['./permission-info.component.scss']
+})
+export class PermissionInfoComponent implements OnInit {
+
+    // the permission info can display the `hasPermission` of a resource and `userHasPermission`together
+    // or only user's permission in case of restricted view
+    @Input() hasPermissions?: string;
+
+    @Input() userHasPermission?: string;
+
+    @ViewChild('infoButton', { static: false }) infoButton: ElementRef;
+
+    // info menu is open or not
+    isOpen = false;
+
+    // default premission values based on DSP-API permissions concept:
+    // https://docs.dasch.swiss/DSP-API/02-knora-ontologies/knora-base/?h=unknown#permissions
+    defaultPermissions: PermissionObj[] = [
+        {
+            name: 'RV',
+            label: 'Restricted view permission (RV)',
+            icon: 'visibility_off' // or disabled_visible or block
+        },
+        {
+            name: 'V',
+            label: 'View permission (V)',
+            icon: 'visibility'
+        },
+        {
+            name: 'M',
+            label: 'Modify permission (M)',
+            icon: 'mode_edit'
+        },
+        {
+            name: 'D',
+            label: 'Delete permission (D)',
+            icon: 'delete'
+        },
+        {
+            name: 'CR',
+            label: 'Change rights permission (CR)',
+            icon: 'admin_panel_settings' // or key
+        }
+    ];
+
+    // default user groups based on DSP-API users and groups concept:
+    // https://docs.dasch.swiss/DSP-API/02-knora-ontologies/knora-base/?h=unknown#users-and-groups
+    defaultGroups: string[] = [
+        'knora-admin:SystemAdmin',
+        'knora-admin:ProjectAdmin',
+        'knora-admin:Creator',
+        'knora-admin:ProjectMember',
+        'knora-admin:KnownUser',
+        'knora-admin:UnknownUser'
+    ];
+
+    listOfPermissions: PermissionGroup[] = [];
+
+    usersRestrictions: PermissionUtil.Permissions[];
+
+    scrollStrategy: ScrollStrategy;
+
+    infoBoxPositions: ConnectionPositionPair[];
+    private _originPos: OriginConnectionPosition = {
+        originX: 'end',
+        originY: 'bottom'
+    };
+
+    private _overlayPos: OverlayConnectionPosition = {
+        overlayX: 'end',
+        overlayY: 'top'
+    };
+
+    constructor(
+        // private _overlay: Overlay,
```

Comment:
this file could use some cleanup (removing commented out code)

Reply:
oops...sure thing &rarr; resolved in fc86cd9b